File: Selenium/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://training-support.net/webelements/login-form")
    print(f"Page title is: {driver.title}")
    username=driver.find_element(By.XPATH,"//input[@name='username']")
    password=driver.find_element(By.XPATH,"//input[@name='password']")
    
    username.send_keys("admin")
    password.send_keys("password")
    login=driver.find_element(By.XPATH,"//button[text()='Submit']")
    login.click
    loginmessage=driver.find_element(By.CSS_SELECTOR,"h1.text-center")
    print("login message is :", loginmessage.text)

File: Selenium/Activity3.java
package activities;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity3 {

	public static void main(String[] args) {
		WebDriver driver=new ChromeDriver();
		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
		driver.get("https://training-support.net/webelements/login-form/");
		System.out.println(driver.getTitle());
		//Find the username field using any locator and enter "admin" into it.
		driver.findElement(By.xpath("//input[@name='username']")).sendKeys("admin");
		//Find the password field using any locator and enter "password" into it.
		driver.findElement(By.xpath("//input[@name='password']")).sendKeys("password");

		//Find the "Log in" button using any locator and click it.
		driver.findElement((By.xpath("//button[text()='Submit']"))).click();
		//Close the browser.
		
		WebElement loginTextElement=driver.findElement(By.xpath("//h2[text()='Welcome Back, Admin!']"));
		String logintext=loginTextElement.getText();
		System.out.println(logintext);
		driver.close();

	}

}
